05-expected-response-match-parser.go

```diff
@@ -64,6 +64,8 @@ func expectedResponseMatchParser(context *context) {
 						"(A|M|N|Y|Z|UT|GMT|[A-Z]{3}|[+-](0\\d|1[012]))"
 			case ":b62:22":
 				reString += "[0-9A-Za-z]{22}"
+			case ":iso8601:µs:z":
```

Comment:
Is it absolutely necessary to use special characters in the tag name? It just makes it that much more difficult to use.